### DISCUSSION TITLE
Could you make the cut, copy, and paste button separate

### DISCUSSION THREAD

**Discussion by Lioncub901** [keiwando/toonsquid#87] [Ideas]
I said before that there was not way to clear a selected area but the cut and paste button seem to be able to do that. The only problem is that is also pastes it so I need to go to the newly created layer and delete it. Could you make the cut, copy, and paste button separate so it will be possible to only cut and selected layer off (could you make the clear button do the same thing like in Procreate).

**Comment by keiwando** [comment]
Yes, I can do that in a future update. Thanks for the feedback!
When it currently pastes, the new layer is already selected, so you can immediately tap the trash button on the right to delete it.

**Comment by Firefly65536** [comment]
I sometimes even use FlipaClip to animate simple stuff just because FlipaClip has the lasso which separates copy and paste, it stores your copied things and you can even paste them across different projects
